homeassistant/components/raspihats/binary_sensor.py

```diff
@@ -54,15 +61,21 @@
 )
 
 
-def setup_platform(hass, config, add_entities, discovery_info=None):
+def setup_platform(
+    hass: HomeAssistant,
+    config: ConfigType,
+    add_entities: AddEntitiesCallback,
+    discovery_info: DiscoveryInfoType | None = None,
+) -> None:
     """Set up the raspihats binary_sensor devices."""
-    I2CHatBinarySensor.I2C_HATS_MANAGER = hass.data[I2C_HATS_MANAGER]
+    I2CHatBinarySensor.I2C_HATS_MANAGER = hass.data[DOMAIN][I2C_HATS_MANAGER]
```

Comment:
It's weird that we attach the manager to the class. I'd pass it to each instance. But it looks ok technically.